Initial modification of how configs are handled for stonesense (#120)
DFHack/stonesense: docs/changelog.txt

## Misc Improvements
- `stonesense`: changed announcements to be right-aligned and limited it to only show the most recent 10 announcements.

> - `stonesense`: `init.txt` config file is now read from ``dfhack-configs/stonesense/init.txt``

Reviewer: ha, whoops.
```
``init.txt``
```

> - `stonesense`: config files are now in `configs/stonesense`in the stonesense repo and are in `dfhack-configs/stonesense` when installed.

Author: oops i forgot it was rst technically

> - `stonesense`: config files are now in ``configs/stonesense``in the stonesense repo and are in ``dfhack-configs/stonesense`` when installed.

Reviewer: ```suggestion
- `stonesense`: `init.txt` config file is now read from ``dfhack-configs/stonesense/init.txt``
```

Reviewer: non-public-facing changes don't need to go in the changelog (unless they have serious consequences for external developers)

> - `stonesense`: creature names are now hidden by default and can be shown by pressing ``n`` with default keybinds.

Reviewer: ```suggestion
- `stonesense`: creature names are now hidden by default (they can still be shown by pressing ``n`` (default keybinding) in stonesense).
```
